src/ui/components/DevTools.tsx

```diff
@@ -50,9 +51,13 @@ function DevTools({
   const userId = claims?.hasura.userId;
 
   const AddSessionUser = hooks.useAddSessionUser();
-  const { recording, isAuthorized, loading: recordingQueryLoading } = hooks.useGetRecording(
-    recordingId
-  );
+  const {
+    recording,
+    isAuthorized,
+    loading: recordingQueryLoading,
+    workspaceName,
+  } = hooks.useGetRecording(recordingId);
+  mixpanel.register({ workspaceName });
```

Comment:
Is this going to complain if the value is `undefined`, like on the recording list page?

Reply:
You're right, not sure what I was thinking there. I guess the question still stands for `undefined` when there is no workspace though.